.circleci/cimodel/data/pytorch_build_definitions.py

```diff
@@ -166,7 +166,8 @@ def gen_workflow_job(self, phase):
                 "branch": self.branch,
                 "requires": [self.parent_build],
                 "context": "org-member",
-                "filters": gen_filter_dict(branches_list=["nightly"])
+                "filters": gen_filter_dict(branches_list=["nightly"],
+                                           tags_list=RC_PATTERN)
```

Comment:
On gotcha that I can foresee potentially happening is that CircleCI has a weird requirement that not all of the jobs can run on a tag, so we'd not only have to add this `gen_filter_dict` call here but we'd need to add it for all of the parent jobs as well. (i.e. all of the jobs that the docs builds `requires`)

Reply:
Ahh, I thought that would resolve automatically. The parent jobs are `pytorch_cpp_doc_build`, `pytorch_python_doc_build`, and the `pytorch-linux-xenial-py3.6-gcc5.4` composite job.